src/app/error.tsx

```diff
@@ -0,0 +1,43 @@
+'use client';
+
+import Link from 'next/link';
+import { useEffect } from 'react';
+
+export default function Error({
+  error,
+  reset,
+}: {
+  error: Error & { digest?: string };
+  reset: () => void;
+}) {
+
+    useEffect(() => {
```

Comment:
I think we can remove all reset related logic from this component since I cannot see a situation where retrying would help resolve the error in our application.